.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Bug report
+about: Create a report to help us fix an issue you experienced.
+labels: bug
+
+---
+
+**Title of the bug** 
+
+*(e.g. Error with `--delete-temp-files`)*
+
+**Description of the bug**
+
+A clear and concise description of what the bug is. 
+
+*(e.g. Used option `--delete-temp-files` did not delete all temporary files after failed transfer)*
+
+**Used versions** (please complete the following information)
+ - Operating System version: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*(e.g. `Windows 10.0.19041`)*
+ - Python version: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*(e.g `3.7.0`)*
+ - PyEGA3 version: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; *(e.g `EGA python client version 3.4.0`)*
```

Comment:
The `&nbsp;` non-breaking spaces look weird here. Should these just be regular spaces?

Reply:
If I'm not mistaken, GitHub's markdown do not allow for tabs within text lines. To make it look nicer I chose to use `&nbsp;`, but it's completely non-essential.

Reply:
I'm fine with the `&nbsp` if the template renders more nicely this way.